### ISSUE TITLE
Update to work with the official Rdio export file now that the service is shutting down?

### ISSUE THREAD

**Issue by iangilman** [shadowhand/radiotide#1]
There aren't any other tools I know of to import into Tidal...it would be great if yours still worked after the shutdown!

**Comment by shadowhand** [comment]
I probably won't work on this project any more. I also synced my data to Spotify and have decided to use that instead of Tidal long term, since Tidal is lacking in a number of key ways for me.

**Comment by iangilman** [comment]
Fair enough. Well, if someone does update it, I'll add it back to http://iangilman.com/rdio-exodus/.

**Comment by iangilman** [comment]
Meanwhile, I've added a "help wanted" to the Developers section on that page.

**Comment by shadowhand** [comment]
Thanks!
